### PR TITLE
Add squashfs-tools to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
This is a package that is needed/used by singularity 3.0

### CODE REVIEW

File: recipes/squashfs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - make install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - lz4-c
+    - lzo
+    - xz
+    - zlib
```
Comment (jakirkham) [inline recipes/squashfs/meta.yaml]: Should these also be listed in `run`?

File: recipes/squashfs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  url: https://downloads.sourceforge.net/project/squashfs/squashfs/squashfs{{ version }}/squashfs{{ version }}.tar.gz
+  sha256: 0d605512437b1eb800b4736791559295ee5f60177e102e4d4ccd0ee241a5f3f6
+  patches:
+    - 0001-modern-compression.diff
+
+build:
+  skip: True  # [not linux]
+  number: 0
+  script:
+    - pushd squashfs-tools
```
Comment (jakirkham) [inline recipes/squashfs/meta.yaml]: If we don't plan to exit this directory, maybe `cd` is sufficient?

**Comment by jakirkham** [comment]
Thanks @sodre. Looks pretty good. Couple comments above.

File: recipes/squashfs/meta.yaml (hunk 1 above)
Comment (sodre) [reply recipes/squashfs/meta.yaml]: It looks like most of those packages already have run_exports in them

  - zlib: https://github.com/conda-forge/zlib-feedstock/blob/58550035ead47ad3e3e45be33d37fbd9965a5cfa/recipe/meta.yaml#L14-L17
  - xz: https://github.com/conda-forge/xz-feedstock/blob/69aecee111a1a0653e21458b88888421e3ae2f34/recipe/meta.yaml#L16-L19
  - lzo: https://github.com/conda-forge/lzo-feedstock/blob/c6fe241568dcaa3226d5b9633d4b2c45ea814b96/recipe/meta.yaml#L14-L16
  - lz4-c: after conda-forge/lz4-c-feedstock#15 is merged.

**Comment by sodre** [comment]
> Thanks @sodre. Looks pretty good. Couple comments above.

Are you okay with me merging it to master once the build passes?

**Pull request merged**